test/e2e/gateway/gateway_test.go

```diff
@@ -212,7 +212,50 @@ var _ = Describe("Gateway API", func() {
 		})
 
 		It("008-tlsroute", func() {
-			testTLSRoute(f)
+			testTLSRoute(f, "gateway-008-tlsroute")
+		})
+	})
+
+	Describe("TLSRoute Gateway: Mode:Terminate", func() {
```

Comment:
should this be passthrough? given https://github.com/projectcontour/contour/pull/3748/files#diff-d6a872e95d486ae59e2bb8b2aaaecf28211105b1c0ec03d65ce18da7a7837290R237

Reply:
yup!